server/docker-compose.yml

```diff
@@ -4,21 +4,12 @@ services:
   ideas-server:
     image: ideasserver
     depends_on: 
+      - wordpress-db
       - initiatives-db
     build:
       context: .
       dockerfile: ideas-server/ideas-server/Dockerfile
 
-  wordpress-db:
```

Comment:
Is the order important?

Reply:
no....the reason is got reordered was because I thought I could take the MSSQL db our of our dockerfile completely, but I ended up putting it back in (in a different spot).
I put it back in because the ServiceEmulator database runs better in MySQL, and the new Log to database also needs a MSSQL db too.

Reply:
Oh gotcha. So in terms of stuff we still have in MSSQL, SB Emulator (only in local dev) and some sort of logging (even for production?)